libxt will fail pkgconf test pipeline

If the pkgconf test pipeline is added to libxt it will fail in the following way:

```
2024/11/12 23:13:01 INFO running step "test/pkgconf"
2024/11/12 23:13:01 WARN + '[' -d /home/build ] uses=test/pkgconf
2024/11/12 23:13:01 WARN + cd /home/build uses=test/pkgconf
2024/11/12 23:13:01 WARN + exit 0 uses=test/pkgconf
2024/11/12 23:13:01 INFO running step "pkgconf build dependency check" uses=test/pkgconf
2024/11/12 23:13:01 WARN + '[' -d /home/build ] uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:13:01 WARN + cd /home/build uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:13:01 WARN + basename /home/build/melange-out/libxt-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:13:01 WARN + dev_pkg=libxt-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:13:01 WARN + cd / uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:13:01 WARN + apk info -L libxt-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:13:01 WARN + grep '\.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:13:01 WARN WARNING: opening /home/user/src/wolfi-os/packages: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:13:01 WARN WARNING: opening from cache https://packages.wolfi.dev/os: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:13:01 WARN + grep -q ^Name: usr/lib/pkgconfig/xt.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:13:01 WARN + basename usr/lib/pkgconfig/xt.pc .pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:13:01 WARN + lib_name=xt uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:13:01 WARN + echo usr/lib/pkgconfig/xt.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:13:01 WARN + grep -q '^usr/lib/pkgconfig/xt.pc$\|^usr/share/pkgconfig/xt.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:13:01 WARN + pkgconf --exists xt uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:13:01 WARN + pkgconf --print-errors --exists xt uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:13:01 WARN Package xproto was not found in the pkg-config search path. uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:13:01 WARN Perhaps you should add the directory containing `xproto.pc' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:13:01 WARN to the PKG_CONFIG_PATH environment variable uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:13:01 WARN Package 'xproto', required by 'xt', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:13:01 WARN Package 'x11', required by 'xt', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:13:01 INFO pod 33c65ddf0412a73a05c9a75f69a552fa304735bb1229121593e5c20ab5d9a28a terminated
2024/11/12 23:13:01 INFO ERROR: failed to test package. the test environment has been preserved:
2024/11/12 23:13:01 INFO   workspace dir: /home/user/tmp/melange-workspace-3530093445
2024/11/12 23:13:01 INFO   guest dir: /home/user/tmp/melange-guest-1181159243
2024/11/12 23:13:01 ERRO failed to test package: unable to run pipeline: unable to run pipeline: unable to run pipeline: task exited with code 1
make: *** [Makefile:147: test/libxt] Error 1
```

--- a/libxt.yaml
+++ b/libxt.yaml
@@ -1,7 +1,7 @@
 package:
   name: libxt
   version: 1.3.1
-  epoch: 0
+  epoch: 1
   description: X11 toolkit intrinsics library
   copyright:
     - license: MIT-open-group
@@ -50,6 +50,9 @@ subpackages:
         - libxt
         - libsm-dev
     description: libxt dev
+    test:
+      pipeline:
+        - uses: test/pkgconf
 
   - name: libxt-doc
     pipeline:

